### PR TITLE
301#3 updated DBStorage and FileStorage

### PR DESCRIPTION
**Pull request by HernanMontenegro**
This is horrible!

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,19 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Retrieve one object"""
+        if (cls is None) or (str(cls) not in classes) or id is None:
```
Comment (ksualcode) [inline models/engine/file_storage.py:74]: `id is None` is not between round brackets, while the other conditions are >:[

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,21 @@ def reload(self):
...
+            return None
+        return self.__session.query(cls).filter_by(cls.id == id)
+
+    def count(self, cls=None):
+        """counts number of objects in storage matching a given class"""
+        if cls is None:
+            pichu = 0
+            for clss in classes.values():
+                pichu += self.__session.query(clss).count()
+            return pichu
```
Comment (ksualcode) [inline models/engine/db_storage.py:91]: `pichu` is not a valid variable name >:[

**Review by ksualcode — CHANGES_REQUESTED** [review]

File: models/engine/file_storage.py (hunk 1 above)
Comment (HernanMontenegro) [reply models/engine/file_storage.py:74]: I will take in consideration your comments, but to be honest, i can't promise anything ;0

**Review by ksualcode — APPROVED** [review]
Great!

**Review by HernanMontenegro — COMMENTED** [review]
Greatn't!

**Pull request closed without merging**